draft-ietf-quic-transport.md

```diff
@@ -4281,8 +4281,8 @@ The BLOCKED frame contains a single field.
 
 Offset:
 
-: A variable-length integer indicating the connection-level offset at which
-  the blocking occurred.
+: A variable-length integer indicating the connection flow control limit at the
+  time the frame is sent.
```

Comment:
I actually prefer the old text, assuming that we do not have a clarification on what "limit" means here. In other parts of the transport protocol we represent the limit using an "inclusive maximum." But here, we need a limit expressed as "exclusive maxmimum."

IMO the original text is clearer in stating that the offset is "maximum offset permitted by MAX_DATA + 1", by saying that the transmitted value is "the offset at which the blocking occurred."